packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1114,6 +1114,14 @@ mixin WidgetInspectorService {
         registerExtension: registerExtension,
       );
 
+      _registerSignalServiceExtension(
```

Comment:
should this be using `registerServiceExtension` instead of `_registerSignalServiceExtension`?

Reply:
I'm not sure, as I copied and pasted this - it looks like `_registerSignalServiceExtension` just calls `registerServiceExtension` but wraps the result properly, so I figured it's a convenience function. Do you know more about what a "signal service" should be?

Reply:
sg to use `_registerSignalServiceExtension` then